PUBLISHING.md

```diff
@@ -0,0 +1,19 @@
+# How to publish releases of elmi-to-json.cabal
+
+
+## Preparation
+
+1. Bump version in package.yaml, package.json, and package-lock.json
```

Comment:
Possibly could use an npm `prepare` hook to automatically update the package.json version from the yaml file.